app/src/main/java/ch/sdp/vibester/user/UserScoreboardAdapter.kt

```diff
@@ -57,10 +58,12 @@ class UserScoreboardAdapter(
          */
         fun bind(player: User, position: Int) {
             val newPosition = position + 1
-            itemView.findViewById<TextView>(R.id.tv_position).text = (newPosition).toString()
+            val rank = (newPosition).toString() + "."
+            itemView.findViewById<TextView>(R.id.tv_position).text = rank
             itemView.findViewById<TextView>(R.id.tv_name).text = player.username
             itemView.findViewById<TextView>(R.id.tv_score).text = setScore(player).toString()
             itemView.findViewById<ImageView>(R.id.iv_photo).loadImg(player.image)
+            if(position %2 == 0) itemView.setBackgroundColor(Color.parseColor("#FFFDF0D7"));
```

Comment:
```suggestion
            if(position %2 == 0) itemView.setBackgroundColor(Color.parseColor("#FFFDF0D7"))
```

Reply:
Plus same remark, you're using two times the same color and it is hardcoded

Reply:
remove colons

Reply:
same as above